lexicons/app/bsky/graph/defs.json

```diff
@@ -0,0 +1,64 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.defs",
+  "defs": {
+    "listViewBasic": {
+      "type": "object",
+      "required": ["uri", "creator", "name", "purpose"],
+      "properties": {
+        "uri": {"type": "string", "format": "at-uri"},
+        "name": {"type": "string", "maxLength": 64, "minLength": 1},
+        "purpose": {"type": "ref", "ref": "#listPurpose"},
+        "avatar": {"type": "string"},
+        "viewer": {"type": "ref", "ref": "#listViewerState"},
+        "indexedAt": {"type": "string", "format": "datetime"}
+      }
+    },
+    "listView": {
+      "type": "object",
+      "required": ["uri", "creator", "name", "purpose"],
+      "properties": {
+        "uri": {"type": "string", "format": "at-uri"},
+        "creator": {"type": "ref", "ref": "app.bsky.actor.defs#profileView"},
+        "name": {"type": "string", "maxLength": 64, "minLength": 1},
+        "purpose": {"type": "ref", "ref": "#listPurpose"},
+        "description": {"type": "string", "maxGraphemes": 300, "maxLength": 3000},
+          "descriptionFacets": {
+            "type": "array",
+            "items": {"type": "ref", "ref": "app.bsky.richtext.facet"}
+          },
+        "avatar": {"type": "string"},
+        "viewer": {"type": "ref", "ref": "#listViewerState"},
+        "indexedAt": {"type": "string", "format": "datetime"}
+      }
+    },
+    "listItemView": {
+      "type": "object",
+      "required": ["subject"],
+      "properties": {
+        "subject": {"type": "ref", "ref": "app.bsky.actor.defs#profileView"},
+        "reason": {"type": "string", "maxLength": 5000, "maxGraphemes": 1000},
+        "reasonFacets": {
+          "type": "array",
+          "items": {"type": "ref", "ref": "app.bsky.richtext.facet"}
+        }
+      }
+    },
+    "listPurpose": {
+      "type": "string",
+      "knownValues": [
+        "app.bsky.graph.defs#modlist"
+      ]
+    },
+    "modlist": {
+      "type": "token",
+      "description": "A list of actors to apply an aggregate moderation on"
```

Comment:
Re: "an aggregate moderation"— is the idea to keep this generic so that it could apply to either blocks or mutes?

Reply:
yup exactly